tests/mongo.yml

```diff
@@ -181,3 +185,18 @@ results:
       - mongo/jstests/aggregation/use_query_project_and_sort.js
       - mongo/jstests/aggregation/use_query_projection.js
       - mongo/jstests/aggregation/use_query_sort.js
+      # uncaught exception: SyntaxError: import declarations may only appear at top level of a module
+      # all these syntax errors are fixed in MongoDB 7.0.
```

Comment:
What does it mean? Isn't that PR updates mongo repo to 7.0?

Reply:
#559 wasn't merged so we were still using the older version of the [shell](https://github.com/FerretDB/dance/blob/main/build/deps/mongo.Dockerfile#L1).

I will approve #559 and then I can remove those failures.

Reply:
(and https://github.com/FerretDB/deps/pull/115)